godotproject/PlayerTongue.gd

```diff
@@ -7,13 +7,18 @@ signal tongue_swing
 export (float) var shoot_speed = 1500 * 0.6
 export (float) var max_shoot_dist = 400 * 0.6
 
+const COLLISION_LAYER_TONGUEABLE 		= 0x3
```

Comment:
why all the tabs?

Reply:
I wanted it to line up pretty with the other layers. More readable the more layers you add

Reply:
ok

Reply:
removed, will be added for taking damge from tongue ticket